lean-client-js-browser/src/inprocess.ts

```diff
@@ -28,9 +49,29 @@ export class InProcessTransport implements Transport {
 
         Module.print = (text: string) => {
             try {
-                conn.jsonMessage.fire(JSON.parse(text));
+                const msg = JSON.parse(text);
+                // replace 'source' fields using olean_map.json if possible
```

Comment:
Nice, is this `olean_map.json` automatically generated?

Reply:
Yeah, it's generated by [the `mk_library.py` script](https://github.com/leanprover-community/lean-web-editor/blob/master/mk_library.py#L95) in leanprover-community/lean-web-editor. (At some point, I should work with Patrick to make that into `leanproject web`.)